syntax = "proto3";
package energy_meter;

import "google/protobuf/timestamp.proto";


service MeasurementService{
  // A simple RPC.
  //
  // Obtains the MessageResponse at a given position.
 rpc GetServerResponse(MeterList) returns (MeterList) {}

}

message Meter {
  google.protobuf.Timestamp time = 1;
  double value = 2;
}

message MeterList {
    google.protobuf.Timestamp start = 1;
    google.protobuf.Timestamp end = 2;
    repeated Meter list = 3;
}

